<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>users_table_entry_template</class>
 <widget class="QWidget" name="users_table_entry_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1020</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 127)</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="username">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="text">
      <string>dummy_username</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="user_id">
     <property name="text">
      <string>dummy_user_id</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="user_post">
     <property name="text">
      <string>dummy_user_post</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="user_special_rights">
     <property name="text">
      <string>dummy_special_rights</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="user_other_data">
     <property name="text">
      <string>dummy_other_data</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
